src/appwrite.ts

```diff
@@ -12,12 +11,11 @@ client
 
 export const createMagicLink = async (id: string, email: string) => {
   try {
-    const a = await account.createMagicURLSession(
+    await account.createMagicURLSession(
       id,
       email,
-      "http://localhost:5173/App"
+      import.meta.env.MAGIC_LINK_URL
```

Comment:
I believe env variables must be prefixed with `VITE_`

https://vitejs.dev/guide/env-and-mode.html#env-files